ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let titleLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    // TODO: Set it as the default image (pawn)
+    @objc public var avatarImage: UIImage? {
+        didSet {
+            if let avatarImage = avatarImage {
```

Comment:
If the image being set is nil, we should honor that and set the avatar view's image to nil too.

Reply:
Good catch. This is definitely needed!